question25/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+img
```

Comment:
あれなんでimgをignoreしているのでしたっけ..?

Reply:
imgってなんだ..

Reply:
I copied the previous assignment per file and it was carried over... I deleted "img" since it is not particularly meaningful this time.

https://github.com/csakyo/homework/pull/39/commits/6fab8e105c1dbb284541373bce679942e58ead73